dependencies.gradle

```diff
@@ -0,0 +1,19 @@
+dependencies {
+    compile("com.github.GTNewHorizons:GT5-Unofficial:experimental-SNAPSHOT:dev") {
+        transitive = false
+        setChanging(true)
+    }
+
+    // GT5U Dependencies
+    compile("net.industrial-craft:industrialcraft-2:2.2.828-experimental:dev")
+    compile("com.github.GTNewHorizons:StructureLib:1.0.6:deobf")
+    compile("codechicken:CodeChickenLib:1.7.10-1.1.3.140:dev")
+    compile("codechicken:CodeChickenCore:1.7.10-1.0.7.47:dev")
+    compile("codechicken:NotEnoughItems:1.7.10-1.0.5.120:dev")
+
+    compileOnly("applecore:AppleCore:1.7.10-1.3.9:deobf")
+    compileOnly("thaumcraft:Thaumcraft:1.7.10-4.2.3.5:dev")
+    compileOnly files("dependencies/HungerOverhaul-1.7.10-1.0.2.jar")
```

Comment:
No need to include a jar file, HungerOverhaul unless you need version 1.0.2:
```groovy
maven {
  name = "hungeroverhaul"
  url = "https://dvs1.progwml6.com/files/maven/"
}
```
and
```groovy
compileOnly("iguanaman:HungerOverhaul:1.7.10-1.0.0.jenkins104:deobf")
```